src/ripple/app/paths/RippleCalc.cpp

```diff
@@ -104,9 +104,7 @@ RippleCalc::Output RippleCalc::rippleCalculate (
         {
             flowV1Out.removableOffers.reserve (
                 rc.permanentlyUnfundedOffers_.size ());
-            std::copy (rc.permanentlyUnfundedOffers_.begin (),
-                rc.permanentlyUnfundedOffers_.end (),
-                std::back_inserter (flowV1Out.removableOffers));
+            flowV1Out.removableOffers = std::move (rc.permanentlyUnfundedOffers_);
```

Comment:
Good call doing the `move`.  With the `move`, is the `reserve` still useful?  I'm guessing you can remove the call to `reserve`.